### ISSUE TITLE
Data pill should expose Last Login Attempt from User-details meta data

### ISSUE THREAD

**Issue by kushalapplaud** [ApplaudSolutions/Applaud-Workato-Connector#47]
## Summary

1. **As a** configurator
1. **I want** to use the Last Login Attempt
1. **so that** I can set validations based on last login

## Acceptance Criteria

1. Last Login Attempt should be accessible in data pills for user search

### [Title of wireframe/mockup 1]
[Image of wireframe/mockup 1]

### [Title of wireframe/mockup 2]
[Image of wireframe/mockup 2]

## Notes

* > Here goes a quote from an email
* Here goes any other useful information…

## Developer Testing
_Requestor:_ leave this section blank
_Developer:_ complete this section during commit/pull request 

**Positive testing:**

Add all the things that you have checked whilst completing this story here, it should be written in a non-technical way, and a lot of this can probably be taken from the acceptance criteria, here are two different examples:  
- **I checked that** the language of the page changed to Spanish **when** I selected Spanish in the language dropdown  
- **I checked that** the list filtered correctly **when** I added an isBefore date filter on list block  

**Negative testing:** 

Here you list the things you have checked where your code needs to show an error, handle unusual/bad input/config gracefully, or may impact separate features that use or depend on the same code. Pause and think -- they may not be obvious!  Examples:    
- **I checked that** there was no "Select language" dropdown option **when** there is only one language configured
- **I checked that** a nice error is shown **when** a language is selected, but the page isn't translated into that language yet
- **I checked that** my list filter code changes didn't cause problems **when** filters are used in other places like ACLs, workflow if/then, and select list filters
- **I checked that** in list filters, I can only select a date type property for the "Value" field, **when** the "Property" field is a date type property

**e2e test links and description:**
- http://www.Link-to-Ghost-inspector-test.com - and paste the test description here
- http://www.Link-to-another-Ghost-inspector-test.com - and paste the test description here

**Comment by kushalapplaud** [comment]
@ssarap Please check and see if cloud enhancement is required for this or this can be handled in SDK itself.

**Comment by duncancasemore** [comment]
@kushalapplaud this sounds like an interesting requirement. Can you provide more background about this?  What do you intend to do with last login attempt if it was available?

**Comment by duncancasemore** [comment]
@kushalapplaud okay, so they basically want to know if a user has signed into Applaud or not?  There's also a user history model (it has things like account verified, password reset, etc) - could that be an option to get this?  I think it may have events that would reveal this although not sure if the APIs are available to API keys or not.

**Comment by duncancasemore** [comment]
@kushalapplaud and speaking to Olivia I was under the impression they know when employees have started or not based on whether they attend their 'IT onboarding' session.  Can that no longer be used?

**Comment by ssarap** [comment]
As Coinbase is not using Workato we can explore option of cloud API.

**Comment by kushalapplaud** [comment]
> @kushalapplaud and speaking to Olivia I was under the impression they know when employees have started or not based on whether they attend their 'IT onboarding' session. Can that no longer be used?

Yes, they are exploring that option through Okta. Actually KCH also had a similar requirement which might come in phase 2 or after go live. There is no urgency for this git right now but these requirements will be coming in future where last login attempt will come in handy.
